examples/multiple-choice/index.es

```diff
@@ -0,0 +1,77 @@
+Element `multiple-choice`
```

Comment:
Can you link me to this in the snuggsi component? I'm not sure how to find this piece.